tools/cli/adminKafkaCommands.go

```diff
@@ -840,28 +840,32 @@ func parseReplicationTask(in string) (tasks []*replicator.ReplicationTask, err e
 	}
 	defer file.Close()
 
-	scanner := bufio.NewScanner(file)
+	reader := bufio.NewReaderSize(file, 1024*1024*500)
```

Comment:
500MB? I think the number is the size for each line in the file right, not the entire file?

Reply:
Consider we have 2mb batch limit. Update to 3MB